replaces torch.Size() inside with a tuple

PursueHappinessDirectly opened the pull request:
Summary: replaces torch.Size() inside with a tuple

Differential Revision: D15460726

colesbury commented:
The reason for this is https://github.com/pytorch/pytorch/issues/20823  -- currently LayerNorm isn't pickleable in Python 2.7

Commit:
replaces torch.Size() inside with a tuple (#20828)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/20828

replaces torch.Size() inside with a tuple

Differential Revision: D15460726

fbshipit-source-id: c75016f1732518be8297e1b403d558a74fbad3d3

colesbury commented:
@driazati landed this in https://github.com/pytorch/pytorch/pull/20832
